rust/macros/helpers.rs

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-2.0
+
+use proc_macro::{token_stream, Group, TokenTree};
+
+pub(crate) fn try_ident(it: &mut token_stream::IntoIter) -> Option<String> {
```

Comment:
Those can just be `pub`

Reply:
That will prevent rustc from giving a warning when they become unused.

Reply:
Why? `mod helpers` is itself private.

Reply:
Yes, but rustc only looks at the visibility modifier on the item itself and doesn't check if there is a public re-export somewhere.

Reply:
I don't think it is bad keeping it `pub(crate)`. So what shall I do?

Reply:
> Yes, but rustc only looks at the visibility modifier on the item itself and doesn't check if there is a public re-export somewhere.

It doesn't seem to be the case. https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=7e3db996c54a04af5b0c2cc23a1ab689

Reply:
You might be confusing dead code check with private type leaking check?

Reply:
I am pretty sure this used to be the case. Guess it was fixed.

Reply:
@bjorn3 is this resolved then?

Reply:
Yes

Reply:
Okay, good to known. Is there something left to change? I sadly lost the overview a bit :-)

Reply:
Well, I still think `pub` should be used here rather than `pub(crate)`.

Reply:
okay, if @bjorn3 is okay with that, I can try to change it tomorrow

Reply:
I am fine either way. My initial concern has been resolved.